### PR TITLE
feat: add rustfmt and clippy

### LINKED ISSUE #9: CI: Update FMT/Clippy
Update CI to run rustfmt and clippy on CI.

### PR DESCRIPTION
**Pull request by asr2003**
/claim #9 
Closes #9

### CODE REVIEW

**Comment by asr2003** [comment]
@tusharmath When I try to regenerate the workflows I am getting this error:
```
\gargantua\wac> npm run generate-workflows

> gaw@1.0.0 generate-workflows
> node ./wac/cli.js generate

node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'C:\Users\91970\Desktop\asr\gargantua\wac\cli.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 { 
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

**Comment by tusharmath** [comment]
Run npm build

**Comment by asr2003** [comment]
@tusharmath Can approve the workflows? I have fixed all clippy issues

File: wac/workflow.ts [hunk 1]
```diff
@@ -1,62 +1,72 @@
...
-      branches: ['main'],
+      branches: ["main"],
     },
     pull_request: {
-      branches: ['main'],
+      branches: ["main"],
     },
   },
-}).addJobs([defaultJob, wasmBuildJob, workflowValidateJob])
+}).addJobs([defaultJob, wasmBuildJob, workflowValidateJob]);
```
Comment (tusharmath) [inline wac/workflow.ts:72]: unnecessary changes in quotes.
